squad-server/plugins/discord-admin-request.js

```diff
@@ -53,13 +53,21 @@ export default class DiscordAdminRequest extends DiscordBasePlugin {
         required: false,
         description: 'The color of the embed.',
         default: 16761867
+      },
+      warnInGameAdmins: {
+        required: false,
+        description:
+          'Should in-game admins be warned after a players uses the command and should we tell how much admins are active in-game right now.',
```

Comment:
We should specify the permission required, i.e. the admin chat.

Reply:
![image](https://user-images.githubusercontent.com/25463237/110180249-b87df500-7e09-11eb-8fd3-f1bca03856eb.png)